LICENSE

```diff
@@ -1,21 +1 @@
-MIT License
-
-Copyright (c) 2023 misikir teka
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
| Your MIT Section | Previous Reviewer MIT Section |
|-|-|
| ![image](https://github.com/misikir21/mama/assets/93039359/26a863ae-517c-495a-b42d-5c8eb5690677) | ![image](https://github.com/misikir21/mama/assets/93039359/2de792e2-8ff1-4763-9da4-100e1e2035fa) |

- [ ] Great work on testing the app successfully but i guess you misunderstood my given change here because i didn't tell you to remove anything from the MIT file 🤔 I just told you that in your readme file your MIT section i am using word section looks like the first image in the above two images 👀 It would be nice if you make this section look like the second image section in your readme file 👍